src/lib/components/atoms/LessonVideo.svelte

```diff
@@ -0,0 +1,20 @@
+<script type="ts">
+	export let videoId: string;
```

Comment:
@IgnacioDebat I believe it's important to make clear that this is a youtube id.
Something like `youtubeVideoId` it's a better name.